### PR TITLE
Bump version to 1.0.0

### PR DESCRIPTION
**Pull request by Warchant**
Signed-off-by: Bogdan Vaneev <warchantua@gmail.com>

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,3 +1,7 @@
+plugins {
+    id 'com.adarshr.test-logger' version '1.6.0'
```
Comment (rogachevsn55) [inline build.gradle]: Why did you choose this logger?

Comment (Warchant) [reply build.gradle]: it is logger for the test output

Comment (rogachevsn55) [reply build.gradle]: Yes, I know. But why don't you use regular logger?

Comment (Warchant) [reply build.gradle]: It just looks better https://github.com/radarsh/gradle-test-logger-plugin

**Pull request merged**